### ISSUE TITLE
method overwritten when I add Turing.jl pkg

### ISSUE THREAD

**Issue by haniehmsv** [TuringLang/Turing.jl#2061]
Hi, when I add Turing.jl and then I type >using Turing, it gives me several warnings related to method overwritten, one is
WARNING: Method definition __init__() in module LinearSolve at C:\Users\Hanieh\.julia\packages\LinearSolve\3dHoj\src\init.jl:1 overwritten at C:\Users\Hanieh\.julia\packages\LinearSolve\3dHoj\src\LinearSolve.jl:120.
  ** incremental compilation may be fatally broken for this module **
And then, this error pops up even when I try to use another package in Julia. 
Do you think this error is related to Turing.jl itself or sth else?

**Comment by devmotion** [comment]
Based on the warnings you posted they seem to be caused by the LinearSolve package.

**Comment by haniehmsv** [comment]
Thank you for the response, but when I completely remove the Turing package, I don't get the warning anymore while using any packages. Do you still think the issue is caused by LinearSolve pkg?

**Comment by devmotion** [comment]
Yes. LinearSolve seems to be an indirect dependency of Turing, therefore you can trigger the issue by loading Turing. The other packages in your environment probably don't depend on LinearSolve.

**Comment by haniehmsv** [comment]
OK, I'll try to reach out to them then. Thank you for your help.

**Comment by Red-Portal** [comment]
I'll close this since this seems to have been resolved.
